packages/xo-server/src/models/user.mjs

```diff
@@ -36,9 +36,13 @@ export class Users extends Collection {
   }
 
   async update(properties) {
```

Comment:
The previous change was incorrect as well.

It was breaking `Collection`'s API: `update()` accepts either a single record or an array of record.

cf. https://github.com/vatesfr/xen-orchestra/blob/34bccbda56b007d7f1eb57a254e37dbd18835a15/packages/xo-server/src/collection.mjs#L70

Reply:
Is there a feature that updates multiple users?

Reply:
Not at this time but it breaks the API, therefore we can expect bugs in the future.

That's why there was a `create()` method instead of overriding `add()`.

I'm working right now on another approach for this.

Reply:
I'm eager we remove this and use SQLite (or other) already.

Reply:
We can reuse the method used with create() which does not override add(), and change the name to update(), for example modify() ?

Reply:
my bad stephane, that was you first approach , and it would not have create this confusion with one / multiple model

Reply:
I feel it's making `Users` more and more different than other collections and complexify the API.

I'm proposing another solution in #8089

Reply:
If I rename update() and modify() in user.mjs, the tests break, probably because update() is called directly.